core/templates/pages/blog-home-page/blog-home-page.component.html

```diff
@@ -88,7 +88,7 @@
           <source type="image/png" [srcset]="getStaticImageUrl('/images/general/no_explorations_found.png')">
           <img [src]="getStaticImageUrl('/general/no_explorations_found.png')">
         </picture>
-        <p class="text-center">
+        <p class="text-center no_result_found">
```

Comment:
Please follow the existing naming convention for class names (we hyphenate them).